src/components/File.jsx

```diff
@@ -69,7 +67,7 @@ class File extends Component {
       this.props.onOpen(id)
     } else {
       // TODO Handle files opening throught the app instead of doing it throught the browser
-      window.open(`${STACK_URL}/files/download/${id}`, '_blank')
+      window.open(`${cozy._url}/files/download/${id}`, '_blank')
```

Comment:
In cozy-client-js, here it is: https://github.com/cozy/cozy-client-js/search?utf8=%E2%9C%93&q=_url

In mobile cozy-files-v3: https://github.com/cozy/cozy-files-v3/search?utf8=%E2%9C%93&q=seturl&type=Code